backend/routers/hf.py

```
@@ -85,6 +89,21 @@ async def hf_chat(req: HFChatRequest):
+async def hf_sentiment(req: HFSentimentRequest):
+    """
+    Analyze sentiment of given text using HF model.
+    """
+    try:
+        result = await analyze_sentiment_hf(req.text)
+        if "error" in result and not result.get("dev_mode"):
+            raise HTTPException(status_code=502, detail=result["error"])
+        return result
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)